src/Opulence/Framework/Tests/Console/Testing/PhpUnit/IntegrationTestCaseTest.php

```diff
@@ -62,9 +62,9 @@ public function testCallingCommandWithMultiplePrompts()
         $this->testCase->execute('multipleprompts', [], [], ['foo', 'bar'])
             ->getResponseAssertions()
             ->outputEquals('Custom1Custom2');
-        $this->testCase->execute('multipleprompts', [], [], ['default1', 'default2'])
```

Comment:
@davidbyoung this is the only test I couldn't really fix, I'm not sure what it's trying to test to be honest.

Reply:
I'm not really sure, either.  We'll leave it off for now.